File: VB/WindowsApplication3/CustomRadioGroup.vb
Imports DevExpress.XtraEditors
Imports System.ComponentModel

Namespace WindowsApplication3

    Public Class CustomRadioGroup
        Inherits RadioGroup

        Shared Sub New()
            RepositoryItemCustomRadioGroup.RegisterCustomEdit()
        End Sub

        Public Sub New()
        End Sub

        Public Overrides ReadOnly Property EditorTypeName As String
            Get
                Return RepositoryItemCustomRadioGroup.CustomEditName
            End Get
        End Property

        <DesignerSerializationVisibility(DesignerSerializationVisibility.Content)>
        Public Overloads ReadOnly Property Properties As RepositoryItemCustomRadioGroup
            Get
                Return TryCast(MyBase.Properties, RepositoryItemCustomRadioGroup)
            End Get
        End Property
    End Class
End Namespace

File: CS/WindowsApplication3/CustomRadioGroup.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;
using DevExpress.Skins;
using DevExpress.XtraEditors;
using System.ComponentModel;

namespace WindowsApplication3 {
    public class CustomRadioGroup : RadioGroup
    {

        static CustomRadioGroup() { RepositoryItemCustomRadioGroup.RegisterCustomEdit(); }

        public CustomRadioGroup() { }

        public override string EditorTypeName { get { return RepositoryItemCustomRadioGroup.CustomEditName; } }

        [DesignerSerializationVisibility(DesignerSerializationVisibility.Content)]
        public new RepositoryItemCustomRadioGroup Properties
        {
            get { return base.Properties as RepositoryItemCustomRadioGroup; }
        }

    }

}